tests/aws/services/events/test_events.py

```diff
@@ -329,12 +338,18 @@ def received(q_urls):
             assert json.loads(msg_received["Body"]) == event
 
         # clean up
-        target_ids = [topic_target_id, sm_target_id, queue_target_id, fifo_queue_target_id]
+        target_ids = [
```

Comment:
just wonder whether these are ruff-related changes or manually triggered 🤔

Reply:
Ruff is not playing nicely with VS Code fixed in.